tensorflow/compiler/tf2tensorrt/convert/utils.cc

```diff
@@ -186,6 +186,25 @@ Status TrtDimsToTensorShape(const nvinfer1::Dims trt_dims,
   return Status::OK();
 }
 
+string TrtDimsToTensorShapeString(const nvinfer1::Dims trt_dims,
```

Comment:
How about using absl::StrCat and absl::StrJoin like here https://github.com/tensorflow/tensorflow/blob/0c79db02d59e7799b3dcf3aff31e0bad71bc809d/tensorflow/compiler/tf2tensorrt/convert/utils.cc#L122-L127
?

Reply:
Oh right. I didn't see that function, my bad.

It's fixed ;)